index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="header">
+      Microverse Supermarket
+      <img
+        src="https://i.pinimg.com/600x315/76/1b/a3/761ba3e1ddf1e70f4cb06906d5935668.jpg"
+        id="header-img"
+        alt="A supermarket clip art"
+      />
+      <br />
+      <br />
+      <br />
+      <br />
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#home">Home Page</a></li>
+          <li><a class="nav-link" href="#about">About</a></li>
+          <li><a class="nav-link" href="#contacts">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+    <h2 id="home">Home</h2>
+    <p>
+      Sit enim labore aute id voluptate Lorem occaecat sint id aute ullamco
+      laborum qui. Duis esse nulla laboris amet tempor incididunt voluptate enim
+      consequat sit adipisicing mollit cupidatat. Id nostrud esse occaecat
+      consectetur magna elit amet ex duis veniam culpa. Esse ad exercitation
+      dolore fugiat ex in. Cupidatat do aliquip anim exercitation consequat
+      laborum est laboris adipisicing. Ullamco in Lorem aliqua nostrud aliqua
+      culpa deserunt dolore aliquip deserunt mollit. Quis amet id deserunt in
+      nisi Lorem dolor incididunt commodo do fugiat ex sunt deserunt. Aute esse
+      aute reprehenderit enim veniam cupidatat amet amet non irure enim et sunt.
+      Eu esse occaecat tempor ea exercitation. Eiusmod exercitation laboris
+      reprehenderit proident dolore pariatur sit enim. Ut cupidatat id id in
+      fugiat. Elit anim non sunt occaecat eiusmod fugiat anim aute excepteur
+      voluptate incididunt. Anim labore nulla sit veniam quis voluptate
+      exercitation non duis deserunt culpa ea aliquip.
+    </p>
+    <h2 id="about">About</h2>
+    <p>Adipisicing ipsum et amet cupidatat.</p>
+    <h2 id="contacts">Contacts</h2>
+    <p>
+      Laborum officia est irure qui incididunt pariatur anim. Ea ullamco
+      reprehenderit duis duis. Ad in esse pariatur aliqua dolore ut non
+      adipisicing elit exercitation consectetur. Non ad sint dolore incididunt
+      dolor.
+    </p>
+    <br />
+    <br />
+    <br />
+    <br />
+    <br />
+    <br />
```

Comment:
[OPTIONAL]  Please use css to style your code. Using [box model](https://www.w3schools.com/css/css_boxmodel.asp) and [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) to style your application gives it a much better feeling to it. It also allows you more flexibility